test/it/user-update.js

```diff
@@ -36,11 +36,12 @@ describe('user.update()', () => {
     _user.profile.nickName = 'rob';
     const modified = JSON.parse(JSON.stringify(_user));
     delete modified.lastUpdated;
+    setTimeout(function () {}, 1000);
```

Comment:
As written that won't defer the execution of your next lines, you'd have to move them into that anonymous function that's inside the setTimeout declaration.

I wrote a delay function in the utils to give this some shorthand, and make it a promise.  I'd try using that.  See usage here:

https://github.com/okta/okta-sdk-nodejs/blob/1546332aae9ee06cb23b8999d58f319a40b445fc/test/it/client-listUsers.js#L35